src/pages/PageStories.js

```diff
@@ -46,18 +49,17 @@ const PageStories = () => {
       </div>
       <div style={styles.scrollText}>
         {/* In order to get rid of the dotted lines, delete "debug" */}
-        <Scrollama onStepEnter={onStepEnter} debug>
+        <Scrollama onStepEnter={onStepEnter} offset={0.5} debug = {true}>
```

Comment:
Delete `= {true}`, as `<Component debug />` is equivalent to `<Component debug={true}>`